README.md

```diff
@@ -1,41 +1,45 @@
-| topic                       | topic                                                        | string |         |                                  | high       |
-| batch.size                  | Number of records to return in a single batch.               | int    | 1024    | [10,...,2147483647]              | medium     |
-| poll.backoff.ms             | The amount of time in ms to wait if no records are returned. | long   | 250     | [10,...,2147483647]              | medium     |
-| dispatcher.thread.pool.size | Number of threads to allocate for the thread pool.           | int    | 10      | [1,...,100]                      | low        |
-| listen.address              | IP address to listen for messages on.                        | string | 0.0.0.0 |                                  | low        |
-| listen.port                 | Port to listen on.                                           | int    | 10161   | ValidPort{start=1025, end=65535} | low        |
-| listen.protocol             | Protocol to listen with..                                    | string | UDP     | [UDP, TCP]                       | low        |
+## General configuration properties
 
+| Name                              | Description                                                   | Type   | Default      | Valid Values                      | Importance |
+|-----------------------------------|---------------------------------------------------------------|--------|--------------|-----------------------------------|------------|
+| topic                             | topic                                                         | string |              |                                   | high       |
+| batch.size                        | Number of records to return in a single batch.                | int    | 1024         | [10,...,2147483647]               | medium     |
+| poll.backoff.ms                   | The amount of time in ms to wait if no records are returned.  | long   | 250          | [10,...,2147483647]               | medium     |
+| dispatcher.thread.pool.size       | Number of threads to allocate for the thread pool.            | int    | 10           | [1,...,100]                       | low        |
+| listen.address                    | IP address to listen for messages on.                         | string | 0.0.0.0      |                                   | low        |
+| listen.port                       | Port to listen on.                                            | int    | 10161        | ValidPort{start=1025, end=65535}  | low        |
+| mpv3.enabled                      | 'true' if mpv3 is enabled                                     | boolean| false        | [true, false]                     | medium     |
 
-# Running in development
 
-The development setup runs with `docker-compose`
+## MPv3 configuration properties
 
-Use
-```
-bin/run.sh
-```
-to build, start the environment and configure the connector using REST API
```

Comment:
Quick instructions to build and start the dev/test env would be preferred over removing everything - including `docker-compose.yml`

Reply:
Yeah but there is no other way to start this at this point, I need to write and create a some kind of env to run it with Strimz for better development experience. Only way to verify at this point is to run tests.